htlcswitch/link_test.go

```diff
@@ -1497,7 +1497,7 @@ func newSingleLinkTestHarness(chanAmt, chanReserve btcutil.Amount) (
 		// to not trigger commit updates automatically during tests.
 		BatchSize:           10000,
 		MinFeeUpdateTimeout: 30 * time.Minute,
-		MaxFeeUpdateTimeout: 30 * time.Minute,
+		MaxFeeUpdateTimeout: 40 * time.Minute,
```

Comment:
Why is 40 more realistic than 30?

Reply:
Since we now do `upper-lower`, this would panic because we're passing in 0 to `math.Rand`.